stdlib/public/Platform/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ add_swift_target_library(swiftDarwin ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_
       ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
       -Xfrontend -disable-objc-attr-requires-foundation-module
     LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
-    TARGET_SDKS ALL_APPLE_PLATFORMS
+    TARGET_SDKS ALL_APPLE_PLATFORMS FREESTANDING
```

Comment:
Maybe the way to do this is to create a notion of if freestanding is "apple" flavored. If it is apple flavored it is part of ALL_APPLE_PLATFORMS and if not. Or we could define a sort of "base" sdk for the freestanding sdk with the idea that it would try to emit these libraries. Just some thoughts.